metrics/system.go

```diff
@@ -32,7 +32,11 @@ func CPUTime() (float64, error) {
 // goroutine.
 func CaptureCPUTime(interval time.Duration) {
 	cpuUtilizationGauge := metrics.NewGaugeFloat64()
-	metrics.Register("runtime.CpuStats.Utilization", cpuUtilizationGauge)
+	err := metrics.Register("runtime.CpuStats.Utilization", cpuUtilizationGauge)
+	if err != nil {
+		log.Warnf("Could not register CPU utilisation metric: %s", err)
```

Comment:
Warn or fatal?

Reply:
I don't see why failing to register a metric should be fatal.